WordPress/src/test/java/org/wordpress/android/ui/posts/prepublishing/visibility/usecases/UpdatePostPasswordUseCaseTest.kt

```diff
@@ -0,0 +1,50 @@
+package org.wordpress.android.ui.posts.prepublishing.visibility.usecases
+
+import com.nhaarman.mockitokotlin2.mock
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.TEST_DISPATCHER
+import org.wordpress.android.fluxc.model.PostModel
+import org.wordpress.android.ui.posts.EditPostRepository
+
+class UpdatePostPasswordUseCaseTest : BaseUnitTest() {
+    lateinit var editPostRepository: EditPostRepository
+    lateinit var updatePostPasswordUseCase: UpdatePostPasswordUseCase
```

Comment:
IDE is complaining these fields could be private.

Reply:
Whoops. Didn't see these warnings. I will address 🙏

Reply:
6514af3